.clang-format

```diff
@@ -7,7 +7,7 @@ IncludeBlocks: Preserve
 NamespaceIndentation: None
 PointerAlignment: Left
 AlignAfterOpenBracket: AlwaysBreak
-AlignEscapedNewlines: DontAlign
+AlignEscapedNewlines: Left
```

Comment:
There you go @momuno. Fixed :)

Reply:
yay!!

Reply:
Nit, the unaligned line continuations are less likely to require future updates.  As it is now the next time you change a line that increases or decreases the maximum width you will need to "unnecessarily" touch each line in the sequence to adjust.

Reply:
@RickWinter, your point is valid, let me know if you think we need to vote. Either way is fine with me.
I did this because our guidelines actually say we should use aligned lines. And I've seen there's a discrepancy between us and the IoT team.
The specific point, about single line update, I do not see as a big problem, because if you change the single line of a macro - you update the entire macro, so I don't see it as a big problem for git history, especially given that normally there aren't that many macros are most of them are up to 5 lines long. On the other hand, single line history is even better, plus it is easier to format by hand, if it comes to that.

Reply:
BTW, all the credit for driving this change and finding this setting goes to @ahsonkhan  - I didn't know this setting existed.

Reply:
@RickWinter - if that becomes too much of a concern, we can use `Right` instead of `Left` for the value of `AlignEscapedNewlines` which moves all the `\` all the way to the right hand side (at around column 100), but still keeping them aligned.

I prefer `Left` though since the macros have relatively small line width, but either work.

Reply:
+1 for Left vs Right alignment for macros (and neutral on DontAlign).